fusor-ember-cli/app/routes/review/installation.js

```diff
@@ -7,8 +8,8 @@ export default Ember.Route.extend({
 
   setupController(controller, model) {
     controller.set('model', model);
-    controller.set('showSpinner', false);
     controller.set('showErrorMessage', false);
+    controller.set('showSpinner', true);
```

Comment:
@daviddavis, I don't see anywhere where you define `spinnerTextMessage`. I just see a spinner with no message.